test/e2e/tests/request-queuing/switchChain-watchAsset.spec.js

```diff
@@ -82,16 +75,21 @@ describe('Request Queue SwitchChain -> WatchAsset', function () {
           tag: 'button',
         });
 
-        await switchToNotificationWindow(driver);
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
 
         // Confirm Switch Network
-        await driver.findClickableElement({
-          text: 'Confirm',
-          tag: 'button',
+        const switchEthereumChainRequestTwo = JSON.stringify({
```

Comment:
Nit: I don't think we need the first switch step anymore. The test case is just that "Add token" confirmations are discarded after switching.

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/27980/commits/a18a31d1c6975dccfc83ece4064ce16464663af3